browser/css/btns.css

```diff
@@ -97,3 +97,12 @@ button.jsdialog img {
 button.has-img {
 	min-width: auto !important;
 }
+
+/* Calc Auto-Filter buttons */
+[data-theme='dark'] button#select_current.has-img > img {
+	filter: invert(.9)
+}
+
+[data-theme='dark'] button#unselect_current.has-img > img {
+	filter: invert(.9)
+}
```

Comment:
Thank you . I would prefer if we would just change the path of src to use the dark subfolder but I see that these are base64 images (probably coming from core) . So I think these lines should get in.